dev/make-distribution.sh

```diff
@@ -163,7 +163,7 @@ echo -e "\$ ${BUILD_COMMAND[@]}\n"
 rm -rf "$DISTDIR"
 mkdir -p "$DISTDIR/jars"
 echo "Spark $VERSION$GITREVSTRING built for Hadoop $SPARK_HADOOP_VERSION" > "$DISTDIR/RELEASE"
-echo "Build flags: $@" >> "$DISTDIR/RELEASE"
+echo "Build flags: "$@"" >> "$DISTDIR/RELEASE"
```

Comment:
@srowen  @jiangxb1987 
I think it's more standardized to use $@ with quotes,this is a blessing and no harm,and we should use $@ with quotes where it will be used.